Apps/W1/HybridGP/app/src/Migration/Accounts/GPAccountMigrator.codeunit.al

```diff
@@ -74,6 +74,65 @@ codeunit 4017 "GP Account Migrator"
         Sender.ModifyGLAccount(true);
     end;
 
+    [EventSubscriber(ObjectType::Codeunit, Codeunit::"GL Acc. Data Migration Facade", 'OnCreateOpeningBalanceTrx', '', true, true)]
+    procedure OnCreateOpeningBalanceTrx(VAR Sender: Codeunit "GL Acc. Data Migration Facade"; RecordIdToMigrate: RecordId)
+    var
+        GPAccount: Record "GP Account";
+    begin
+        if RecordIdToMigrate.TableNo() <> Database::"GP Account" then
+            exit;
+
+        GPAccount.Get(RecordIdToMigrate);
```

Comment:
Should we have if not GPAccount.Get(RecordIdToMigrate) then 
                               exit;

Just to be on the safe side?

Reply:
There appears to be an issue with the merge I did from main as there was only to be a single line changed. I believe we covered this same question in a previous PR (https://github.com/microsoft/ALAppExtensions/pull/19202). Perhaps I was mistaken as to when that PR was going into BC. This PR is only needed to fix that PR.